swagger.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
I would suggest you a YAML file rather JSON as YAML is far more easy to write and read than JSON.

Reply:
Ok, thanks  I have will into it now